terraform/modules/jenkins/ec2.tf

```diff
@@ -3,14 +3,14 @@ provider "aws" {
 }
 
 resource "aws_instance" "jenkins3" {
```

Comment:
Does this still need to be called jenkins3?

Reply:
Possibly not actually. I previously thought it wasn't going to be able to change these names, but have since learned that it might. I'll have a go.

Reply:
Yep, it was totally possible. All done.

Reply:
Nice!